docusaurus/docs/React/hooks/message-hooks.mdx

```diff
@@ -179,6 +179,26 @@ const MyCustomMessageComponent = () => {
 };
 ```
 
+### useReactionsFetcher
+
+A [custom hook](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Message/hooks/useReactionsFetcher.ts) to handle loading message reactions. Returns an async function that loads and returns message reactions.
```

Comment:
Would it make sense to mention there is a limit how many reactions are returned at max? (`MAX_MESSAGE_REACTIONS_TO_FETCH`)